spec.md

```diff
@@ -5,76 +5,56 @@ keywords: registry, on-prem, images, tags, repository, distribution, api, advanc
-  - [Pulling An Image](#pulling-an-image)
-  - [Pushing An Image](#pushing-an-image)
-  - [Listing Repositories](#listing-repositories)
-  - [Listing Image Tags](#listing-image-tags)
-  - [Deleting an Image](#deleting-an-image)
-- [Detail](#detail)
-  - [Errors](#errors-2)
-  - [Base](#base)
-  - [Tags](#tags)
-  - [Manifest](#manifest)
-    - [GET Manifest](#get-manifest)
-    - [PUT Manifest](#put-manifest)
-    - [DELETE Manifest](#delete-manifest)
-  - [Blob](#blob)
-    - [GET Blob](#get-blob)
-      - [Fetch Blob](#fetch-blob)
-      - [Fetch Blob Part](#fetch-blob-part)
-    - [DELETE Blob](#delete-blob)
-  - [Initiate Blob Upload](#initiate-blob-upload)
-    - [POST Initiate Blob Upload](#post-initiate-blob-upload)
-      - [Initiate Monolithic Blob Upload](#initiate-monolithic-blob-upload)
-      - [Initiate Resumable Blob Upload](#initiate-resumable-blob-upload)
-      - [Mount Blob](#mount-blob)
-  - [Blob Upload](#blob-upload)
-    - [GET Blob Upload](#get-blob-upload)
-    - [PATCH Blob Upload](#patch-blob-upload)
-    - [PUT Blob Upload](#put-blob-upload)
-    - [DELETE Blob Upload](#delete-blob-upload)
-
-## Notational Conventions
-
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" are to be interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997).
-
-The key words "unspecified", "undefined", and "implementation-defined" are to be interpreted as described in the [rationale for the C99 standard][c99-unspecified].
-
-An implementation is not compliant if it fails to satisfy one or more of the MUST, MUST NOT, REQUIRED, SHALL, or SHALL NOT requirements for the protocols it implements.
-An implementation is compliant if it satisfies all the MUST, MUST NOT, REQUIRED, SHALL, and SHALL NOT requirements for the protocols it implements.
-
-## Historical Context
+The Open Container Initiative Distribution Specification defines an API protocol to facilitate and standardize the distribution of content, especially related to container images and associated artifacts.
+
+### Historical Context
+
+The spec is based on the specification for the [Docker Registry HTTP API V2 protocol](https://github.com/docker/distribution/blob/5cb406d511b7b9163bff9b6439072e4892e5ae3b/docs/spec/api.md).
 
 For relevant details and a history leading up to this specification, please see the following issues:
 
 - [moby/moby#8093](https://github.com/moby/moby/issues/8093)
 - [moby/moby#9015](https://github.com/moby/moby/issues/9015)
 - [docker/docker-registry#612](https://github.com/docker/docker-registry/issues/612)
 
-<!--- TODO: add relevant background information here --->
+### Definitions
+
+Several terms are used frequently in this document and warrant basic definitions:
+
+- **Registry**: a HTTP service which implements this spec
```

Comment:
s/HTTP//

Reply:
`a service which implements the server side of this spec`? because technically the client implements the spec too

Reply:
>a service that handles the required APIs in this specification

Reply:
handles (the server side of) ? or is that repetitive?